apps/api/src/app/inbox/utils/notification-mapper.ts

```diff
@@ -0,0 +1,73 @@
+import type { MessageEntity } from '@novu/dal';
+import { ButtonTypeEnum } from '@novu/shared';
+
+import type { InboxNotification, Subscriber } from './types';
+
+const mapSingleItem = ({
+  _id,
+  content,
+  read,
+  archived,
+  createdAt,
+  lastReadDate,
+  archivedAt,
+  channel,
+  subscriber,
+  actorSubscriber,
+  actor,
+  cta,
+  tags,
+}: MessageEntity): InboxNotification => {
+  const to: Subscriber = {
+    id: subscriber?._id ?? '',
+    firstName: subscriber?.firstName,
+    lastName: subscriber?.lastName,
+    avatar: subscriber?.avatar,
+    subscriberId: subscriber?.subscriberId ?? '',
+  };
+  const primaryCta = cta.action?.buttons?.find((button) => button.type === ButtonTypeEnum.PRIMARY);
+  const secondaryCta = cta.action?.buttons?.find((button) => button.type === ButtonTypeEnum.SECONDARY);
+
+  return {
+    id: _id,
+    // subject: subject,
+    body: content as string,
+    to,
+    read,
+    archived,
+    createdAt,
+    readAt: lastReadDate,
+    archivedAt,
+    actor: actorSubscriber
+      ? {
+          id: actorSubscriber._id,
+          firstName: actorSubscriber.firstName,
+          lastName: actorSubscriber.lastName,
+          avatar: actorSubscriber.avatar,
+          subscriberId: actorSubscriber.subscriberId,
+        }
+      : undefined,
+    avatar: actor,
```

Comment:
Maybe a better name for this param, as it's a bit confusing?

Reply:
just the `icon`?